app/tasks/raster_tile_cache_assets/symbology.py

```diff
@@ -548,7 +548,7 @@ async def _create_intensity_asset(
     )
 
     if zoom_level == max_zoom:
-        parent_jobs: Optional[List[Job]] = None
+        parent_jobs: List[Job] = [jobs_dict[zoom_level]["source_reprojection_job"]]
```

Comment:
I haven't spent much time looking at `tasks`, yet. Is this how we create processing pipelines? Does this schedule work to be done or describe a dependency for the child jobs? 🤷

Reply:
It describes a dependency for the child jobs.
We probably could handle the dependencies a little better than a dictionary. For the moment, look at https://github.com/wri/gfw-data-api/blob/upgrade_packages_tests/app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py#L120-L180 to better understand what's happening. 
To see where the actual submission to AWS Batch happens, look at [gfw-data-api/app/tasks/batch.py](https://github.com/wri/gfw-data-api/blob/upgrade_packages_tests/app/tasks/batch.py)

Reply:
Thanks for the links, @dmannarino!